spec/requests/historical_banner_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+RSpec.describe "Documents" do
+  describe "GET /documents/:document" do
+    describe "history mode banner" do
+      context "when in history mode" do
+        it "shows the history mode banner" do
+          @edition = create(:edition, :published, :political, :past_government)
```

Comment:
I don't think you need :published here or in the other places where we have it in this test.

Reply:
sorted!